### DISCUSSION TITLE
code=45, title=禁止登录, message=你的帐号涉嫌违规被限制在非常用设备登录，请更换常用设备后重新登录并根据提示进行解封。

### DISCUSSION THREAD

**Discussion by Pika-Lee** [cssxsh/fix-protocol-version#30] [Announcements]
今天登陆的时候报这个错误了，我用自己的手机QQ登录被冻结账号，仍然弹窗提示“请更换常用设备后重新登录并根据提示进行解封。”

请问下大家遇到这种情况应该如何解除账号冻结呢？谢谢。

**Comment by cssxsh** [comment]
把 `bot/.../` 删掉

**Reply by Pika-Lee** [reply]
试了一下还是不行
![QQ截图20230702041656](https://github.com/cssxsh/fix-protocol-version/assets/23114990/acba966a-ab1a-4e6c-9bba-eaa2a7dc90d0)

**Comment by cssxsh** [comment]
确认 1845605751 已解封

**Reply by Pika-Lee** [reply]
我在aq.qq.com/007查了一下还是冻结状态
![QQ截图20230702042448](https://github.com/cssxsh/fix-protocol-version/assets/23114990/e6c78e2d-769f-4b20-b88a-a1e0c5b0627f)

**Reply by Pika-Lee** [reply]
我尝试用自己的手机QQ登录被冻结账号，仍然弹窗提示“请更换常用设备后重新登录并根据提示进行解封。”
![QQ截图20230702043115](https://github.com/cssxsh/fix-protocol-version/assets/23114990/55ed3b66-4f24-40b6-b8c4-75032f33fbc5)

**Comment by Pika-Lee** [comment]
补充一条，我换了其它QQ出现了同样的账号冻结问题。无法使用号主的手机QQ登录账号解封（被判定为“非常用设备”）
![QQ截图20230702194005](https://github.com/cssxsh/fix-protocol-version/assets/23114990/3270ce08-b26e-4559-a4e3-a24144a7f89e)
![QQ截图20230702194135](https://github.com/cssxsh/fix-protocol-version/assets/23114990/dcfc71b9-16ed-490f-a584-ef4d2ea588bb)

**Comment by gebederry** [comment]
我的情况和你相同，提示的意思大致是 可以去申请解冻，但同时还在提示你非常用设备。
实际上是哪个设备都无法登录，都会提示你非常用设备。
貌似无解
暂时收手不折腾了，等疼讯心情好了再说

**Reply by Pika-Lee** [reply]
我今天试了一个号，在我的手机QQ上成功登陆并解冻，但其它几个号在手机QQ上依然提示“非常用设备”，目前卡在这里了

**Comment by gebederry** [comment]
> 我今天试了一个号，在我的手机QQ上成功登陆并解冻，但其它几个号在手机QQ上依然提示“非常用设备”，目前卡在这里了
同，之前一直提示非常用的一个号，昨天试着登录允许解冻了。似乎有个期限，一个月。
